### ISSUE TITLE
unable to fetch and push 

### ISSUE THREAD

**Issue by brysonwaisi** [atom/github#2434]

**Comment by rahulsps** [comment]
This is similar to https://github.com/atom/github/issues/2430

**Comment by saiv123** [comment]
yes i have having a simillar problem, so everything working fine except when it is trying to push or pull the url it is trying to pull from is something like github.com/username/repo.git/ when its supposed to be github.com/username/repo.git
